src/CGOptions.cpp

```diff
@@ -199,6 +199,7 @@ DEFINE_GETTER_SETTER_BOOL(fast_execution);
 //GCC C Extensions
 DEFINE_GETTER_SETTER_BOOL(func_attr_inline);
 DEFINE_GETTER_SETTER_BOOL(func_attr_aligned);
+DEFINE_GETTER_SETTER_BOOL(function_attributes);
```

Comment:
I would rather name it `function_attr_boolean` or `function_attr_flag` to express that it's a single true/false attribute.

Reply:
> I would rather name it `function_attr_boolean` or `function_attr_flag` to express that it's a single true/false attribute.
Done. Changed name to "func_attr_flag"